scss/utilities/_clearfix.scss

```diff
@@ -1,3 +1,11 @@
 .clearfix {
   @include clearfix();
 }
+
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-only($breakpoint) {
```

Comment:
> But should the normal .clearfix be renamed to .clearfix-xs-up?

IMHO, yes. That's what we did with the responsive `.pull-*-*` classes. It avoids ambiguity, inconsistency, and redundancy (albeit at slight cost in learning curve).